src/authorizer/authorizer.lambda.ts

```diff
@@ -0,0 +1,34 @@
+
+export async function handler(event: any):Promise<any> {
+
+  // Step 1: Extract the token from the Authorization header
+  const token = event.authorizationToken;
+
+  // Log the received token for debugging purposes
+  console.log('Received Token:', token);
+
+  // Step 2: Validate if the token is exactly "dummy"
+  if (token === 'dummy') {
```

Comment:
Deze moet in productie ook niet kunnen denk ik

Reply:
Als niet in gebruik is deze comment overbodig

Reply:
Niet in gebruik